staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -682,6 +682,8 @@ message LinuxPodSandboxStats {
     ProcessUsage process = 4;
     // Stats of containers in the measured pod sandbox.
     repeated ContainerStats containers = 5;
+    // Swap usage gathered for the pod sandbox.
+    SwapUsage swap = 6;
```

Comment:
actually: are there pod level stats for cadvisor? or pod level swap defined at all? it's possible we actually don't need this, and can just rely on the sum(ContainerStats.swap..)

Reply:
That sounds right :+1: 
Thanks, done